assets/js/lib/network/index.js

```diff
@@ -68,36 +68,36 @@ function handleResponseStatus(response) {
   return response;
 }
 
-export const post = function post(url, data) {
+export const post = function post(url, data, config) {
```

Comment:
We can pass the config to overwrite the existing one, in our case the `baseURL`. 
If an empty config is provided, the default one is used:
https://axios-http.com/docs/instance

Reply:
Do you think that for readability we could have the config with a default argument of null or something at least compatible with the "empty config" just to match the arity of the function without explicitly ignore the argument